tests/scripts/remote_monitoring_tests/rest_apis/test_update_recommendations.py

```diff
@@ -305,34 +305,5 @@ def test_update_recommendations_with_unknown_experiment_name_and_end_time(cluste
     response = update_recommendations(experiment_name, None, end_time)
     data = response.json()
     assert response.status_code == ERROR_STATUS_CODE
-    assert data['message'] == UPDATE_RECOMMENDATIONS_DATA_NOT_FOUND
+    assert data['message'] == UPDATE_RECOMMENDATIONS_DATA_NOT_FOUND%(experiment_name)
 
-
-@pytest.mark.negative
```

Comment:
we can consider this test when interval_start_time is exposed to consumers.

Reply:
You can skip the test with this marker -  @pytest.mark.skip(reason="reason")